addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1,4 +1,7 @@
 # Kodi Media Center language file
+# [Capitalization] Avoid capitalizing every second word e.g. "This New String", unless proper name or proper noun.
```

Comment:
we also need a rule on how to refer to a possible select value of a spinner when referring to it in the settings description. I think we now sometimes use "settings value" and sometimes [settings value]. IMO we should use braces for this case and quotes when referring to other stuff.